frontend/public/index.html

```diff
@@ -5,6 +5,26 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/static/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+    <!-- for security -->
+    <meta http-equiv="Content-Security-Policy" 
+          content="default-src 'self';
+                   connect-src 'self' https://api.mapbox.com https://cdn.jsdelivr.net https://dev.virtualearth.net https://events.mapbox.com https://osm-stats-production-api.azurewebsites.net https://osmstats-api.hotosm.org https://taginfo.openstreetmap.org https://tasking-manager-tm4-production-api.hotosm.org https://www.openstreetmap.org; 
```

Comment:
I think it's quite difficult to maintain that list of urls, as we can use new APIs, and other instances will use others too. I observed that even other Mozilla websites don't have it, like getpocket.com, getfirefox.com, developer.mozilla.org

Reply:
These URL are not arbitrary, we simply list the connections that are made from within tasking-manager. Our code must have this list. Apart from this, the various maps that OSM uses make a small list.